packages/core/src/tools/experimentalFeatures.ts

```diff
@@ -19,6 +22,16 @@ export enum ExperimentalFeature {
 
 const enabledExperimentalFeatures: Set<ExperimentalFeature> = new Set()
 
+export function initFeatureFlags(enableExperimentalFeatures?: string[] | undefined) {
+  if (Array.isArray(enableExperimentalFeatures)) {
+    addExperimentalFeatures(
+      enableExperimentalFeatures.filter((flag): flag is ExperimentalFeature =>
+        objectHasValue(ExperimentalFeature, flag)
+      )
```

Comment:
*Thoughts:* we don't really need this validation, we could save a few bytes by removing it

Reply:
Yes, I was also wondering why we had it. I'll remove it

Reply:
Hum, it seems there is a test made on purpose https://github.com/DataDog/browser-sdk/blob/77596277f4c9e70f55eb4e373323b63e22f7ccc7/packages/core/src/tools/experimentalFeatures.spec.ts#L46